cdap-common/bin/functions.sh

```diff
@@ -413,7 +413,20 @@ cdap_set_hbase() {
     1.2-cdh*) __compat="${CDAP_HOME}"/hbase-compat-1.2-cdh5.7.0/lib/* ;; # 5.7 and 5.8 are compatible
     1.2*) __compat="${CDAP_HOME}"/hbase-compat-1.1/lib/* ;; # 1.1 and 1.2 are compatible
     "") die "Unable to determine HBase version! Aborting." ;;
-    *) die "Unknown/Unsupported HBase version found: ${HBASE_VERSION}" ;;
+    *-cdh*)
+      if [[ $(cdap_get_conf "hbase.version.resolution.strategy" "${CDAP_CONF}"/cdap-site.xml auto.strict) == 'auto.latest' ]]; then
+        __compat="${CDAP_HOME}"/hbase-compat-1.2-cdh5.7.0/lib/* # if allowed, return the latest CDH HBase version for unsupported HBase version
```

Comment:
See note below, but in either case, add two trailing semi-colons (preceded by a space, as other statements):
```
__compat="${CDAP_HOME}"/hbase-compat-1.2-cdh5.7.0/lib/* ;;